src/MICore/Transports/UnixShellPortTransport.cs

```diff
@@ -17,17 +18,24 @@ public class UnixShellPortTransport : ITransport, IDebugUnixShellCommandCallback
         private string _startRemoteDebuggerCommand;
         private IDebugUnixShellAsyncCommand _asyncCommand;
         private bool _bQuit;
+        private bool _debuggerLaunched = false;
+        private HostWaitLoop _waitLoop;
+
+        private static string ErrorPrefix = "Error:";
```

Comment:
No

---
In reply to: [75037013](https://github.com/Microsoft/MIEngine/pull/414#discussion_r75037013) [](ancestors = 75037013)